app/imports/client/ui/creature/character/CharacterSheet.vue

```diff
@@ -186,6 +187,7 @@ export default {
     },
   },
   mounted() {
+  setUpDddiceSdk();
```

Comment:
This call sets up the dddice sdk, inserts the canvas, and creates a connection to the dddice servers.

There is probably a way better place to call this from, I'm just not sure where is the correct place.  For the sake of the POC i thought this was good enough.